packages/website/package.json

```diff
@@ -1,102 +1,95 @@
 {
   "name": "@web3-storage/website",
-  "version": "1.10.0",
+  "version": "2.0.0",
   "description": "web3.storage website",
   "private": true,
   "scripts": {
-    "start": "next dev -p 4000",
-    "build": "next build && next-sitemap && next export",
-    "test": "eslint './**/*.js' && tsc --build"
+    "dev": "rm -rf .next && next dev -p 4000",
+    "start": "next start",
```

Comment:
Please make `npm start` run the website in dev mode. We don't use an npm run script for production, so we optimise for developers here and make `npm start` do the thing.